src/EA.Weee.Web/Areas/Admin/Controllers/ReportsController.cs

```diff
@@ -443,6 +448,46 @@ public async Task<ActionResult> MissingProducerData(MissingProducerDataViewModel
             return File(data, "text/csv", CsvFilenameFormat.FormatFileName(fileData.FileName));
         }
 
+        [HttpGet]
+        public async Task<ActionResult> AatfAeReturnData()
+        {
+            SetBreadcrumb();
+            ViewBag.TriggerDownload = false;
+
+            AatfAeReturnDataViewModel model = new AatfAeReturnDataViewModel();
+            await PopulateFilters(model);
```

Comment:
Can we move this method into a view model builder, will make the code tidier and easier to test